src/scripts/cfo.zig

```diff
@@ -73,6 +73,48 @@ const Fuzzer = enum {
     vsr_journal_format,
     vsr_superblock_quorums,
     vsr_superblock,
+    vopr,
+    vopr_testing,
+
+    fn print_command(fuzzer: Fuzzer, shell: *Shell, seed: u64) ![]const u8 {
+        if (fuzzer == .vopr or fuzzer == .vopr_testing) {
+            const state_machine: []const u8 =
+                if (fuzzer == .vopr) "" else "-Dsimulator-state-machine=testing";
+            return try shell.print(
+                "./zig/zig build -Drelease{s} simulator_run -- {d}",
+                .{ state_machine, seed },
+            );
+        }
+        return try shell.print(
+            "./zig/zig build -Drelease fuzz -- --seed={d} {s}",
+            .{ seed, @tagName(fuzzer) },
+        );
+    }
+
+    fn spawn_command(fuzzer: Fuzzer, shell: *Shell, seed: u64) !std.ChildProcess {
+        if (fuzzer == .vopr or fuzzer == .vopr_testing) {
+            const state_machine: []const []const u8 =
+                if (fuzzer == .vopr) &.{} else &.{"-Dsimulator-state-machine=testing"};
+            return try shell.spawn_options(
+                .{ .stdin_behavior = .Pipe },
+                "{zig} build -Drelease {state_machine} simulator_run -- {seed}",
+                .{
+                    .zig = shell.zig_exe.?,
+                    .state_machine = state_machine,
+                    .seed = seed,
+                },
+            );
+        }
+        return try shell.spawn_options(
+            .{ .stdin_behavior = .Pipe },
+            "{zig} build -Drelease fuzz -- --seed={seed} {fuzzer}",
```

Comment:
I don't love how I have to repeat essentially the same thing again and again and again and again here. I'll solve this separately, by harmonizing vopr and minor fuzzer's CLI interface (I think I'll take VOPR's approach of passing the seed in as a positonal)